dkg-gadget/src/worker.rs

```diff
@@ -416,10 +416,10 @@ where
 						lock[async_index as usize] = Some(status_handle)
 					} else if current_round.is_active() {
```

Comment:
As mentioned in our meeting, the `is_active()` will actually "mostly" return `true` since singing rounds stuck in `OfflineAndVoting` stage.